### ISSUE TITLE
Include the 'Belt' standard library

### ISSUE THREAD

**Issue by austindd** [rebench/rebench.github.io#7]
Since BuckleScript officially ships with `Belt` as an alternative standard library (and officially recommends its use), `Belt` should be available in ReBench.

I am not sure how to go about integrating it at this time, but it would be a very welcomed improvement!

**Comment by glennsl** [comment]
This uses [`bs-in-a-box`](https://github.com/glennsl/bs-in-a-box) and is just a matter of replacing the [vendored compiler and stdlib bundle](https://github.com/glennsl/bs-in-a-box/tree/master/vendor) there as long as the interface is still the same. Unfortunately I suspect it's not. I also think the process of building it has changed a lot.

I think @ryyppy and @thangngoc89 know the process best these days. Maybe they can chime in, or have plans to do something similar to this elsewhere.

**Comment by thangngoc89** [comment]
@glennsl I totally forgot about rebench, i’ll see if I can update this

**Comment by tsnobip** [comment]
I agree, rebench is so useful to quickly benchmark BS code, having access to Belt would be a game changer because Belt implementation usually makes it much faster than the JS lib and a bit faster than Stdlib.

I don't know if things have become a bit easier to integrate since last year, I know there's some ongoing work with a new playground for [reasonml.org](reasonml.org), maybe some of the efforts could be reused here.
